yagna_dapp_manager/dapp_manager.py

```diff
@@ -52,8 +52,12 @@ def start(
         storage.init()
 
         starter = DappStarter(descriptor_paths, config_path, storage)
-        starter.start()
+        pid, stdout, stderr = starter.start(timeout=1)
```

Comment:
This is now an argument to `DappManager.start()` because:
* This will make #45 super-easy
* This way we can easily pass a 0-timeout in tests (where we want to, and we usually do)